tensilelite/Tensile/SolutionStructs.py

```diff
@@ -3031,11 +3031,12 @@ def assignDerivedParameters(state):
       storeInstMinWidth = 1 # minimum dwordx1
       storeInstMaxWidth = 4 # maximum dwordx4
       srMinVw = max(storeInstMinWidth, int(storeInstMinWidth/state["ProblemType"]["DestDataType"].numRegisters()))
-      numReg  = state["ProblemType"]["DestDataType"].numRegisters()
-      if state["_GlobalAccumulation"]:
-        numReg = state["ProblemType"]["ComputeDataType"].numRegisters()
+      numReg  = max(state["ProblemType"]["DestDataType"].numRegisters(), state["ProblemType"]["ComputeDataType"].numRegisters())
 
       srMaxVw = int(storeInstMaxWidth/numReg)
+      # FIXME: Add StoreRemapVectorWidthGSU and StoreRemapVectorWidthNonGSU
+      while srMaxVw < state["StoreRemapVectorWidth"]:
+        state["StoreRemapVectorWidth"] = state["StoreRemapVectorWidth"] // 2
```

Comment:
you add a loop here, so you can remove the following line 
" or srMaxVw < state["StoreRemapVectorWidth"] "
because it will not happen